### DISCUSSION TITLE
render() view()->title() triggers error

### DISCUSSION THREAD

**Discussion by CyberPunkCodes** [livewire/livewire#7429] [7. Bugs (archived)]
### Livewire version

3.2.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

```php
<?php

namespace App\Livewire;

use Livewire\Component;

class Dashboard extends Component
{
    public function render()
    {
        return view('livewire.pages.dashboard')->title('Dashboard');  // VSCode error on title()
    }
}
```

### Additional information

At least VSCode shows an error on the `title()` method chained on the `view()`.

Is it possible to fix? I know simple doc blocks can resolve this on other areas. Just not sure if it's possible with the way Livewire does things.

### Screenshot:

<img width="630" alt="Screen Shot 2023-12-03 at 3 31 48 AM" src="https://github.com/livewire/livewire/assets/9557392/b947f9b2-0dba-4739-865a-33de750f46ba">

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by wishborn** [comment]
I get this problem as well and it's very annoying. Makes me waste so much time going to check and see if I have an error.
